src/BackgroundLayerChooser/BackgroundLayerChooser.tsx

```diff
@@ -28,12 +28,14 @@ import './BackgroundLayerChooser.less';
 
 export type BackgroundLayerChooserProps = {
   layers: OlLayer[];
+  showEmptyLayer?: boolean;
```

Comment:
We probably want to have a more saying name here. Maybe something along the lines of `allowNoBackground`, `allowEmptyBackground`, ...?

Reply:
I have changed it to `allowEmptyLayer`.

Reply:
I have changed it to `allowEmptyBackground`.